Helper/OmiseHelper.php

```diff
@@ -276,6 +276,36 @@ public function omiseAmountFormat($currency, $amount)
         return $amount;
     }
 
+    /**
+     * @param  string  $currency
+     * @param  integer $amount
+     *
+     * @return string
+     */
+    public function currencyAmountFormat($currency, $amount)
+    {
+        $supportedCurrencies = [
```

Comment:
Did you miss to add Japan currency here? This code `return $amount /= 100;` will not work for Japanese currency.